core/import_data/import_progress_reports.py

```diff
@@ -0,0 +1,124 @@
+                    agency,
+                ]
+            ):
+                continue
+
+            data = {
+                "project": project,
+                "project_type": project_type,
+                "sector": project_sector,
+                "status": project_status,
+                "latest_status": latest_status,
+                "agency": agency,
+                "country": country,
+                "region": region,
+                "chemical": item["chemical"],
+                "mtg": item["mtg"],
+                "num": item["num"],
+                "a_n": item["a_n"],
+                "irdx": item["irdx"],
+                "consumption_odp_out_proposal": item["consumption_odp_out_proposal"],
+                "consumption_odp_out_actual": item["consumption_odp_out_actual"],
+                "production_odp_out_proposal": item["production_odp_out_proposal"],
+                "production_odp_out_actual": item["production_odp_out_actual"],
+                "date_approved": parse_date(item["date_approved"], logger),
+                "date_first_disbursement": parse_date(
+                    item["date_first_disbursement"], logger
+                ),
+                "date_comp_proposal": parse_date(item["date_comp_proposal"], logger),
+                "date_comp_actual": parse_date(item["date_comp_actual"], logger),
+                "funds_approved": item["funds_approved"],
+                "funds_adjustment": item["funds_adjustment"],
+                "funds_net": item["funds_net"],
+                "funds_disbursed": item["funds_disbursed"],
+                "percent_disbursed": item["percent_disbursed"],
+                "balance": item["balance"],
+                "funds_obligated": item["funds_obligated"],
+                "funds_current_year": item["funds_current_year"],
+                "support_approved": item["support_approved"],
+                "support_adjustment": item["support_adjustment"],
+                "support_disbursed": item["support_disbursed"],
+                "support_obligated": item["support_obligated"],
+                "support_returned": item["support_returned(37only)"],
+                "year_approved": item["year_approved"],
+                "months_first_disbursement": item["months_first_disbursement"],
+                "months_comp_proposal": item["months_comp_proposal"],
+                "months_comp_plan": item["months_comp_plan"],
+                "months_comp_actual": item["months_comp_actual"],
+                "remarks_1": item["remarks_1"],
+                "remarks_2": item["remarks_2"],
+                "year_of_contribution": item["year_of_contribution"],
+                "BP_allocation": item["BP_Allocation"],
+                "category": item[
+                    "multi-year/one-off phaseout/individual/rmp/rmp update"
+                ],
+                "meeting_of_report": item["meeting_of_report"],
+            }
+            ProjectProgressReport.objects.create(
```

Comment:
hmmm, we might end up using this command a bunch of times on the same database. We should make sure that we don't create any duplicates if we run it again

Reply:
It may prove difficult to find an unique identifier here. Will try to figure something out :sweat_smile: